Patterns/LayeredProject.md

```diff
@@ -0,0 +1,100 @@
+# Layered project
+
+Keeping the structure of each .NET project somewhat similar helps developers to quickly find existing code and directly know where to add new code.
+
+## Solution layout
+
+A .NET solution must follow the following guidelines:
+
+- The name of the solution is `{Customer}.{Project}.sln`.
+- Put simple, solution-wide used objects like enums, exceptions and simple classes in Core: `{Customer}.{Project}.Core.csproj`.
```

Comment:
Not 'Common'?

Reply:
I like 'core' as a word... but it is a bit ambiguous with the framework itself now :/

Reply:
Agreed, +1 for using Common.

Reply:
Will fix